scripts/resume_arm_time.py

```
@@ -56,6 +56,45 @@
 
 _HELPER_PATH = os.path.realpath(__file__)
 
+
+def _is_under_temp(path):
+    temp_roots = ("/tmp", "/var/tmp", "/private/var/folders", "/var/folders")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)